package.json

```diff
@@ -25,16 +25,19 @@
     "dist/jquery.validate.unobtrusive.min.js"
   ],
   "dependencies": {
-    "jquery": "^3.5.1",
-    "jquery-validation": ">=1.16"
+    "jquery": "^3.6.0",
+    "jquery-validation": ">=1.19"
   },
   "devDependencies": {
     "gulp": "^4.0.2",
     "gulp-line-ending-corrector": "^1.0.3",
-    "gulp-rename": "1.2.2",
-    "gulp-replace": "0.5.4",
-    "gulp-uglify": "2.0.0",
-    "lodash": "^4.17.20",
-    "minimatch": ">=3.0.2"
+    "gulp-rename": "2.0.0",
+    "gulp-replace": "1.1.3",
+    "gulp-uglify": "3.0.2",
```

Comment:
Why are these specific versions❔ Suggest at least allowing patch bumps the next time someone runs `npm install` e.g. `"gulp-rename": "~2.0.0"`.

Reply:
I didn't pay too much attention to these, as the tooling did it's magic here. I'll update these too.